podcast-player/podcast-player-app/src/components/episode/EpisodePlayer.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+function EpisodePlayer({ episode }) {
+  // const { $length, $type, $url } = content;
+  return (
+    <div className="player">
+      <audio
+        controls
+        src={episode.enclosure.$url}
```

Comment:
If we're already massaging the parsed XML data, we should probably also transform the episode data into a more useable shape.